### PR TITLE
127 stripe checkout session

### PR DESCRIPTION
**Pull request by jeffplays2005**
Created a method to create a new stripe checkout session.

### CODE REVIEW

File: server/src/business-layer/services/StripeService.ts [hunk 1]
```diff
@@ -21,4 +21,24 @@ export default class StripeService {
     })
     return result.data
   }
+
+  public async createCheckoutSession(
+    client_reference_id: string,
+    return_url: string,
+    line_item: any,
```
[the remark this answers is not in the document]
Comment (bcho892) [reply server/src/business-layer/services/StripeService.ts:28]: ignore that, it should be of a type given in the example from stripe docs
```ts
line_items: [
    {
      price: 'price_1MotwRLkdIwHu7ixYcPLm5uZ',
      quantity: 2,
    },
  ],
```

so `{price: string, quantity: number}`

**Review by bcho892 — COMMENTED** [review]
Just some small changes :)

**Comment by jeffplays2005** [comment]
Fixed type errors. Thank you Benson!

**Review by bcho892 — APPROVED** [review]
lgtm! Closes #127

**Pull request merged**